bin/check-licenses.mjs

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+import { spawnSync } from 'node:child_process';
+
+/**
+ * Internal dependencies
+ */
+import { checkDepsInTree } from '../packages/scripts/utils/license.js';
+
+const ignored = [ '@ampproject/remapping' ];
```

Comment:
I see it’s licensed under [Apache-2.0 license](https://github.com/ampproject/remapping/blob/main/LICENSE) so it would be handy to add now a comment explaining why it can be ignored. I hope it’s only listed as a dependency of a dependency but never used.

Reply:
So this is a dependency of `@babel/core`. The only production package that depends on `@babel/core` that I could locate is `@emotion/css`. It's used in a single place:

https://github.com/WordPress/gutenberg/blob/b8cd03d2e2713ffe648923712d2599b25808ad67/packages/components/src/utils/hooks/use-cx.ts#L7-L10

It doesn't seem to use Babel at all:

> `cx` is emotion's version of the popular [`classnames` library](https://github.com/JedWatson/classnames).

The latest version of `@emotion/css` doesn't define `@babel/core` as a peer dependency anymore:

https://github.com/emotion-js/emotion/blob/main/packages/css/package.json

Reply:
That would be nice, but I'd prefer to leave upgrades for after this PR has landed. It's already quite a big and potentially disruptive PR.

Reply:
I wanted to document everything as I was digging into the details of prior decisions. There is no need for any follow-up in this PR.